data/brands/office/consulting.json

```diff
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^consulting$"]}
   },
   "items": [
+    {
+	    "displayName": "Kelly Services",
+	    "locationSet": {"include": ["001"]},
+	    "tags": {
+	      "brand": "Kelly Services",
+	      "brand:wikidata": "Q1538585",
+	      "name": "Kelly Services",
+	      "office": "consulting"
```

Comment:
I've search by overpass-api and found many Kelly Services are tagged with `office = employment_agency`, will that tag bettter? (Which value do you think is more suitable to describe Kelly Services? Which kind of services does it provide? I'd suggest you search on osmwiki, and make it clear whether the company's main business is "providing job assistance to individuals" or "providing business management strategy assistance to enterprises")

Reply:
Hi,

This company provides a lot of services from Financial, IT, law and Educational etc. They say their main business is consulting and outsourcing - https://www.kellyservices.com/global/about-us/company-information/company-overview2/#:~:text=Today%2C%20we%20offer%20a%20full%20suite%20of%20outsourcing%2C%20consulting%2C%20and%20staffing%20solutions%E2%80%94delivered%20globally
That said `office = employment_agency` will suit better given its services. I will close this one and move it to `office = employment_agency`.

Thanks,